rcfiles/jointLklDM.rc

```diff
@@ -35,14 +35,18 @@ jointLklDM.plotsDir: plots
 # Comment out if you don't want to save or read the histograms but create them on the fly
 # (that is slower but safer because you make sure you use the proper histograms and not
 #  some pre-existing version created for a different set of IRFs or samples)
-jointLklDM.dNdEpSignalDir: DM/dNdEpSignal
+# Comment out for the channel 'branon' for now!!!
+# Comment out for the channel 'custom' always!!!
```

Comment:
I don't think these two comments are necessary, for branon the total dN/dE is fixed right, so it makes perfect sense to allow writing the resolution-convoluted dNdEp histograms. 
Custom should not be a channel, but rather a given "br1*channel1+...+brn*channeln" string, which I think can be used in the name of the dNdEp file without any problem (maybe some "\" commands have to be included, no big deal...)

Reply:
This comments are remains from the time I used the 'custom' as a channel keyword. I will delete them.